tests/helpers.py

```diff
@@ -39,6 +44,66 @@ def create_unique_resource_name(resource_description, resource_type):
     return f"{resource_type}_{resource_description[:23]}_{current_date_time[:10]}"
 
 
+def create_resource(
+    resource_kind, resource_name, project_name, yaml_file, **kwargs
```

Comment:
maybe template_yaml instead of yaml_file?

Reply:
done